data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/chance_to_hit/chance_to_hit_two_cumulative_no_same_id.cfg

```diff
@@ -0,0 +1,38 @@
+# wmllint: no translatables
+
+#####
+# API(s) being tested: [chance_to_hit]cumulative=
+##
+# Actions:
+# Give the leaders a chance_to_hit ability with non-unique IDs and cumulative=no
+# Have the side 1 leader attack the side 2 leader with both its weapons
+##
+# Expected end state:
+# The side 1 leader's first weapon doesn't strike 100 times
+# The side 1 leader's second weapon doesn't strike 100 times
+#####
+{COMMON_KEEP_A_B_UNIT_TEST "chance_to_hit_two_cumulative_no_same_id" (
+    [event]
+        name = start
+
+        {SET_HP VALUE=1000}
+
+        [modify_unit]
+            [filter]
+            [/filter]
+            [effect]
+                apply_to = new_ability
+                [abilities]
+                    {TEST_ABILITY attacks 100 () SELF=yes}
+                    {TEST_ABILITY chance_to_hit 6 () ID=1 CUMULATIVE=no SELF=yes}
+                    {TEST_ABILITY chance_to_hit 4 () ID=1 CUMULATIVE=no SELF=yes}
```

Comment:
Should these be 60 and 40? With them currently adding to only 10%, it doesn't seem to matter whether they're cumulative or not, Alice isn't going to hit 100/100 strikes.

Applies to a few other tests too.

Reply:
Updated.